packages/rs-dpp/src/document/document_factory/v0/mod.rs

```diff
@@ -330,7 +330,7 @@ impl DocumentFactoryV0 {
         documents
             .into_iter()
             .map(|(document, document_type)| {
-                if document_type.documents_mutable() {
+                if document_type.documents_read_only() {
```

Comment:
You are doing the opposite...

Reply:
Why are we changing this to documents_read_only... there are so many places in the code where we might overlook something now.